backend/build.gradle

```diff
@@ -15,16 +15,26 @@ version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
 
 ext {
-    kickstartVersion = '11.0.0'
-    set('springCloudVersion', "2020.0.5")
+    set('springCloudVersion', "2021.0.3")
 }
 repositories {
     mavenCentral()
 }
 
+jar {
```

Comment:
new spring generates two jar files, which messes up the ci pipeline since it expects one jar in the `build/libs `dir
this removed the extra jar